crates/packager/src/config/mod.rs

```diff
@@ -889,7 +893,12 @@ impl Config {
 
     /// Returns the out dir
     pub fn out_dir(&self) -> PathBuf {
-        dunce::canonicalize(&self.out_dir).unwrap_or_else(|_| self.out_dir.clone())
+        if self.out_dir.as_os_str().is_empty() {
+            // TODO: we should probably error out when the out dir isn't set
```

Comment:
using the current directory (which will be the directory containing the configuration) here is good enough IMO,